editors/vscodium.nix

```diff
@@ -0,0 +1,27 @@
+{ pkgs ? import <nixpkgs> { } }:
+with pkgs;
+let
+  codium = vscode-with-extensions.override {
+    vscode = vscodium;
+    vscodeExtensions = with vscode-extensions; [
+      jnoortheen.nix-ide
+    ];
+  };
+in
+writeShellScriptBin "codium-test" ''
+  set -e
+  dir="''${XDG_CACHE_HOME:-~/.cache}/nixd-codium"
+  ${coreutils}/bin/mkdir -p "$dir/User"
+  cat >"$dir/User/settings.json" <<EOF
+  {
+  "security.workspace.trust.enabled": false,
+  "nix.enableLanguageServer": true,
+  "nix.serverPath": "nixd",
+  }
+  EOF
+  if [[ "$XDG_SESSION_TYPE"=="wayland" ]];then
+      NIXOS_OZONE_WL=1 ${codium}/bin/codium --disable-gpu --user-data-dir "$dir" "$@"
```

Comment:
Is this works for all wayland users? I'm concerning about some people on wayland may not launch it correctly, with wayland ozone. Maybe just use `${codium}/bin/codium --user-data-dir "$dir" "$@"`, (i.e. use xwayland) will be better.

Reply:
I think what you say is reasonable, it may not work for all wayland users, if there is a problem, just leave the launch parameter to the user to decide